service/gcs/core/gcs/gcs.go

```diff
@@ -56,25 +56,23 @@ func NewGCSCore(rtime runtime.Runtime, os oslayer.OS) *gcsCore {
 // containerCacheEntry stores cached information for a single container.
 type containerCacheEntry struct {
 	ID                 string
-	ExitStatus         oslayer.ProcessExitState
```

Comment:
I'm not committed to using `exitCode` vs `ExitStatus` but nobody ever uses it for anything other than the code so it seemed easier to only cache the code